apps/hash-external-services/docker-compose.yml

```diff
@@ -72,11 +73,17 @@ services:
         condition: service_healthy
       telemetry-collector:
         condition: service_healthy
-    build:
+    build: &graph-build
       dockerfile: docker/Dockerfile
       context: ../../apps/hash-graph
       args:
         PROFILE: dev
+    image: hash-graph
```

Comment:
Do we need to specify image as well? I thought htis would conflict with `build`? This goes for the other services that have this

Reply:
This ensures that we use the exact same image for all three of the services, it's just the name of the image.